flask/test/auth_tests/test_auth.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+from auth import get_user_token_string
+
+from boxwise_flask.auth_helper import (  # ,; decode_jwt,; requires_auth,; 
+    user_can_access_base,
+    authorization_test,
+    get_rsa_key,
+    get_token_from_auth_header,
+    AuthError,
+    requires_auth,
+    decode_jwt
+)
+
+def test_get_valid_jwt():
+    token = get_token_from_auth_header(get_user_token_string())
+    key = get_rsa_key(token)
+    assert key != None
```

Comment:
is there maybe a better assertion then != None?
Does the key have a specific length or format?

Reply:
exceptions are thrown in the cases where this is not true so if you get to this stage then none is okay